src/kibana/components/courier/saved_object/saved_object.js

```diff
@@ -265,6 +269,24 @@ define(function (require) {
         });
       };
 
+      self.export = function () {
+        return {
+          _id: this.id,
+          _type: type,
+          _source: this._source
+        };
+      };
+
+      self.import = function (result) {
+        self.id = result._id;
+        Object.keys(result._source).forEach(function (key) {
+          self[key] = result._source[key];
+        });
+
+        parseSearchSource(self.kibanaSavedObjectMeta && self.kibanaSavedObjectMeta.searchSourceJSON);
+
+        return self.save();
```

Comment:
Does `self.save()` update `self._source`? Noticed that `self._source` is not updated at the top of this function.